.circleci/scripts/release-bump-changelog-version

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+set -o pipefail
+
+if [[ "${CI:-}" != 'true' ]]
+then
+    printf '%s\n' 'CI environment variable must be set to true'
+    exit 1
+fi
+
+if [[ "${CIRCLECI:-}" != 'true' ]]
+then
+    printf '%s\n' 'CIRCLECI environment variable must be set to true'
+    exit 1
+fi
+
+version="${CIRCLE_BRANCH/Version-v/}"
+
+if grep --quiet --fixed-strings "$version" CHANGELOG.md
+then
+    printf '%s\n' 'Adding this release to CHANGELOG.md'
+    date_str="$(date '+%a %b %d %Y')"
+    # Replace '## Current Develop Branch\n\n' with "## Current Develop Branch\n\n## ${version} ${date_str}\n\n"
```

Comment:
I'm guessing a command should be here to perform this replacement?

Reply:
Whoops, yes

Reply:
What command would you suggest for this?

Reply:
Maybe something like `sed -i'' '/## Current Develop Branch/c## [replacement title]' CHANGELOG.md `?

Reply:
Yeah I have something similar locally but it's not quite correct:

```
sed -i .bak -e "s/## Current Develop Branch/## Current Develop Branch\\n\\n## ${version} ${date_str}/" CHANGELOG.md
```

Reply:
I'm exploring solutions using AWK now

Reply:
Got a solution with AWK